optimum/onnxruntime/modeling_diffusion.py

```diff
@@ -666,4 +696,140 @@ class ORTStableDiffusionXLImg2ImgPipeline(ORTStableDiffusionXLPipelineBase, Stab
     ONNX Runtime-powered stable diffusion pipeline corresponding to [diffusers.StableDiffusionXLImg2ImgPipeline](https://huggingface.co/docs/diffusers/api/pipelines/stable_diffusion/stable_diffusion_xl#diffusers.StableDiffusionXLImg2ImgPipeline).
     """
 
+    main_input_name = "prompt"
+    auto_model_class = StableDiffusionXLImg2ImgPipeline
+
     __call__ = StableDiffusionXLImg2ImgPipelineMixin.__call__
+
+
+SUPPORTED_ORT_PIPELINES = [
+    ORTStableDiffusionPipeline,
+    ORTStableDiffusionImg2ImgPipeline,
+    ORTStableDiffusionInpaintPipeline,
+    ORTLatentConsistencyModelPipeline,
+    ORTStableDiffusionXLPipeline,
+    ORTStableDiffusionXLImg2ImgPipeline,
+]
+
+
+def _get_pipeline_class(pipeline_class_name: str, throw_error_if_not_exist: bool = True):
+    for ort_pipeline_class in SUPPORTED_ORT_PIPELINES:
```

Comment:
for the case where export=False, we should look for diffusers classes name instead no ? (removing the OV prefix)

Reply:
i'm not sure I understand, why when export=False ? the `ORTDiffusionPipeline` only dispatches to one of the classes based on `_class_name`

Reply:
In the case where the model needs to be exported (so when export=True not export=False my bad) we should look for diffusers classes instead of the ORT ones

Reply:
The underlying ORT class takes care of the export, these classes only dispatch to one of them (to one of the task and architecture specific ORT classes).

Reply:
ok actually I see :
```
or ort_pipeline_class.auto_model_class.__name__ == pipeline_class_name
```
which is for the export=True case, so should work indeed!